new-frontiers-14/frontier-station-14
# Mercfab recipes

## About the PR
Allows the mercfab to print Arsenal equipment.
## Why / Balance
With the exception of ship guns, more specifically the grenade launchers, most Arsenal tech does not have enough destructive power in space to warrant gating it to only NFSD.

## How to test

Load a server
Spawn a Mercfab, observe the rubber rounds
Do some arsenal research, observe the arsenal tech added to the fab.

## Media
Incendiary and uraniuum munitions
![image](https://github.com/user-attachments/assets/eaf78151-5cd2-4d2f-b399-122cd084f1b9)

## Requirements

## Breaking changes
Nothing I can see.
**Changelog**
:cl:
- The Mercfab is able to print arsenal technology, excluding ship guns and grenade launcher shells.
